crates/l2/sequencer/mod.rs

```diff
@@ -49,13 +50,13 @@ pub async fn start_l2(
     {
         error!("Error starting Committer: {err}");
     };
+    if let Err(err) =
+        ProofCoordinator::spawn(store.clone(), rollup_store.clone(), cfg.clone()).await
+    {
+        error!("Error starting Proof Coordinator: {err}");
+    };
```

Comment:
What do you think of this?
```suggestion
    let _ = ProofCoordinator::spawn(store.clone(), rollup_store.clone(), cfg.clone())
        .await
        .inspect_err(|err| {
            error!("Error starting Proof Coordinator: {err}");
        });
```

Reply:
Done